validation/schema.json

```diff
@@ -230,7 +230,8 @@
                       "type": {
                         "enum": [
                           "SEMVER",
-                          "ECOSYSTEM"
+                          "ECOSYSTEM",
```

Comment:
we can just get rid of the entire allOf, since SEMVER, ECOSYSTEM, and GIT encompass every single type.

Reply:
Good point, removed